README.md

```diff
@@ -1 +1,195 @@
-# react-solo-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>Rest Countries 🌎</b></h1>
+
+</div>
+
+# Table of Contents
+
+- [ Table of Contents](#table-of-contents)
+- [🌎 Space Travelers' Hub ](#about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [`npm start`](#npm-start)
+    - [`npm test`](#npm-test)
+    - [`npm run build`](#npm-run-build)
+    - [`npm run eject`](#npm-run-eject)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Rest Countries <a name="#about-project"></a>
+
+Explore the world with **Rest Countries**, a mobile web app that offers an engaging and user-friendly way to discover information about countries. Browse a wide range of nations, then dive deeper into each one's details with just a tap. From population to languages spoken, Rest Countries provides a visually appealing and organized presentation of essential country insights. Start your global journey today!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+    <li><a href="#">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Displaying list of all country around the world.
+- Seach bar for searching specific country.
+- display list of each country the country that we click.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] According to the previous reviewer and requirements:

![image](https://github.com/mahdinoori2000/react-solo-capstone/assets/77069011/cb4d2739-65f4-4c5e-b511-f5fd30fb13c4)

Please consider deploying your project! 🤓 🔧